nd4j/nd4j-backends/nd4j-backend-impls/nd4j-native/src/main/java/org/nd4j/linalg/cpu/nativecpu/bindings/Nd4jCpu.java

```diff
@@ -478,6 +478,13 @@ public native void copyBufferFrom(@Const @ByRef DataBuffer other, @Cast("size_t"
 
   public native void setPrimaryBuffer(Pointer buffer, @Cast("size_t") long length);
   public native void setSpecialBuffer(Pointer buffer, @Cast("size_t") long length);
+// #ifndef __JAVACPP_HACK__
+// #endif
+
+  public native void showBufferLimited();
+  //for Debug purposes
+  public native void showCounters(@Cast("char*") String msg1, @Cast("char*") String msg2);
+  public native void showCounters(@Cast("char*") BytePointer msg1, @Cast("char*") BytePointer msg2);
```

Comment:
Were these changes supposed to make it into this pull request?

Reply:
Yes these are part of Rauf's VEDA PR.